src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Packages.props

```diff
@@ -19,29 +19,35 @@
     <!--#endif-->
     <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
     <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.2.221116.1" />
```

Comment:
Can we update this to [1.2.221209.1](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.2.221209.1)